external-crates/move/crates/move-compiler/src/expansion/ast.rs

```diff
@@ -742,6 +742,10 @@ impl Visibility {
             Visibility::Internal => None,
         }
     }
+
+    pub fn is_public(&self) -> bool {
+        matches!(self, Visibility::Public(_))
+    }
```

Comment:
Let's remove this, we don't typically write these sort of predicates for straight-foward matches like this

Reply:
Should we then make the `Visibility` variants public? That was the reason I couldn't do.